views/components/PageHeader/index.jsx

```diff
@@ -1,8 +1,48 @@
 import React from 'react'
-import { Box, Typography, useMediaQuery } from '@mui/material'
+import { Box, Typography, IconButton } from '@mui/material'
+import { ArrowBack } from '@mui/icons-material'
+import { fontSize } from '../../../constants'
 
 const PageHeader = (props) => {
-  return (
+  return props.isDescription ? (
+    <Box
+      sx={{
```

Comment:
I continue to dislike `sx` from a separation of concerns perspective. This is a comment on `mui` not your code, your implementation is the recommended way to use the framework. Maybe in a future project these `sx` objects could go in their own files, but I dunno I just don't like styles in JS.

Reply:
ya I know it's weird and it has it's own properties and nested objects